logprep/framework/pipeline.py

```diff
@@ -267,6 +267,8 @@ def process_pipeline(self) -> PipelineResult:
                 self.logger.error(",".join((str(error) for error in result.errors)))
                 self._store_failed_event(result.errors, result.event_received, event)
                 return
+        else:
+            return PipelineResult(results=[], event=event, pipeline=[], event_received=event)
```

Comment:
```suggestion
```

instead  consider initialize result with None after line 259

Reply:
and we should tests this. it was not a good idea, to implement this without tests. sorry... shame on me.